commands/custom/clearAndDelete.js

```diff
@@ -0,0 +1,16 @@
+const clearAndDelete = client => async ({ Bucket }) => {
+  const res = await client.listObjectsV2({ Bucket });
+
+  if (res.Contents) {
```

Comment:
We need to create an issue to improve this command in the future so iterate over the results (if the bucket has too many objects, it returns a paginated response)

Reply:
Ticket added